docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
 
     elasticsearch:
         image: elasticsearch
+        ports:
+            - "9200:9200"
```

Comment:
This looks like something that should be used temporarily for debugging and not on the public master branch. Please remove